internal/ark/config.go

```diff
@@ -275,6 +278,23 @@ func (req ConfigRequest) Get() (values ValueOverrides, err error) {
 		},
 		InitContainers: initContainers,
 		CleanUpCRDs:    true,
+		Affinity: map[string]interface{}{
+			"nodeAffinity": map[string]interface{}{
+				"requiredDuringSchedulingIgnoredDuringExecution": map[string]interface{}{
+					"nodeSelectorTerms": []map[string]interface{}{
+						{
+							"matchExpressions": []map[string]interface{}{
+								{
+									"key":      "kubernetes.io/arch",
+									"operator": "In",
+									"values":   []string{"amd64"},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
```

Comment:
**Question**: what is the reason behind favoring node affinity over [node selector](https://github.com/banzaicloud/velero-helm-chart/blob/main/charts/velero/values.yaml#L54)?

As long as the restricting expression is a single node label equaling a single value (and currently the set of values allowed has a single item) my understanding is that they are equivalent solutions (CMIIW), but the affinity is significantly more complex to define, although it is in fact easier to extend in the future should the need arise.

Reply:
Node selector should be enough.

Reply:
In this simple case node selector should be fine, however it's generally is better to use affinity instead since it's newer flexible and node selector might be deprecated in future.

Reply:
Hopefully we can remove this in a couple months.

Reply:
I'm fine with either solution, just wanted to know if there is an objective reason to prefer affinity over selector in case of interchangability.